docs/topics/Gateway.md

```diff
@@ -1053,6 +1056,18 @@ Sent when a guild role is deleted.
 | guild_id | snowflake | id of the guild |
 | role_id  | snowflake | id of the role  |
 
+### Guild Scheduled Event Create
+
+Sent when a guild event is created. The data contain a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object.
```

Comment:
```suggestion
Sent when a guild event is created. The inner payload is a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object.
```
This and the Update and Delete descriptions should be consistent with the others.